### PR TITLE
support top & left arrow offsets together

### LINKED ISSUE #34: allow Arrow to offset both left and top
seems somewhat unnecessary to only support left OR top offset for arrow, but not both: https://github.com/souporserious/react-popper/blob/master/src/Popper.jsx#L155-L159

I'm writing a modifier to dynamically offset the popper on the alternate axis and counter-offset the arrow on the same axis, instead of relying on CSS margins etc, but react-popper actively fights this.

Is it fair to just do the simple thing and simply apply `top` and `left` offsets at all times?

### PR DESCRIPTION
**Pull request by giladgray**
fixes #34

### CODE REVIEW

File: src/Popper.jsx [hunk 1]
```diff
@@ -152,11 +152,7 @@ class Popper extends Component {
       return {}
     } else {
       const { top, left } = this.state.data.offsets.arrow
-      if (!left) {
-        return { top: +top }
-      } else {
-        return { left: +left }
-      }
+      return { top: +top, left: +left }
```
Comment (giladgray) [inline src/Popper.jsx]: 🤔 @FezVrasta should I include bottom and right here too? feels like a yes.

**Comment by FezVrasta** [comment]
I'm not at pc but I thought the computeStyle modifier created an object with any needed style prop just like it happens for the popper element. This way you don't need to manually specify any

**Comment by giladgray** [comment]
@FezVrasta `computeStyle` does not produce an object for arrow, only for popper
https://github.com/FezVrasta/popper.js/issues/347

File: src/Popper.jsx (hunk 1 above)
Comment (giladgray) [inline src/Popper.jsx]: nvm, not necessary

**Comment by FezVrasta** [comment]
My lib needs some refactor/cleaup so bad.. 🙄

**Comment by giladgray** [comment]
ok @souporserious i think this PR is sufficient for now. 
the next iteration will wait for https://github.com/FezVrasta/popper.js/issues/347.

**Comment by giladgray** [comment]
@souporserious what's the path to merging this and #40? i'll need a release containing both of them to unblock some of my other feature work. happy to help however I can!

**Comment by souporserious** [comment]
Sorry been super busy this last week. Looking at everything now :) thanks for all of your work! I appreciate it.

**Pull request merged**